docs/application/web/get-started/addon/first-addon.md

```diff
@@ -0,0 +1,521 @@
+   }
+   ...
+   ```
+
+   You can see that the `loadURL` has been moved from `resume()` to `myPrelaunch()`.
+
+2. Finding the way to return to the main page of the Web application
+
+   Now, we can control the frequent loading of the addon's HTML page. However, we are not able to return to the main page of the Web application.
+
+   How about if we can get the URL of the main page of the Web application ? We have the way !
+
+   In the parameter of `myPrelaunch()` above, you can find `orig`. That has the URL of the main page of the Web application. You need to keep & use it when you need. If you want to return to the main page when it goes to the suspend state, i.e. `lcSuspend` event. The implementation will be like:
+
+   ```
+   constructor(em) {
+       ...
+       this.org = null;
+   }
+   activate() {
+       ...
+       this.evt.on('lcSuspend', this.returnToMain);
+       ...
+   }
+   deactivate() {
+       ...
+       this.evt.off('lcSuspend', this.returnToMain);
+       ...
+   }
+   myPrelaunch(winId, orig) {
+       this.org = orig;
+       ...
+   }
+   resume(winId) {
+       ...
+   }
+   returnToMain(winId) {
+       addonAPI.window.loadURL(winId, this.org);
+   }
+   ```
+
+   You can test the implementation by the sequences as below :
+
+   1. In the Web application, press Home key to enter the suspend state
+   2. Long-press the Home key and select Basic UI to enter the resume state
+
+### Addition : Return to main page of Web application by back key
+
+In showing `hello.html` with previous solution, there will be no reaction when you press the Back Key.
+To resolve it, you have to change `hello.html` as following:
+
+```
+<head>
+    <meta name="viewport" content="user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0, width=device-width">
+    <script type="text/javascript">
+        window.onload = function() {
+            document.addEventListener("tizenhwkey", function(e) {
+                if (e.keyName == "back") try {
+                    // your action
+                } catch (t) {}
+            });
+        }
+    </script>
+</head>
+```
+
+Now, you can put your code in the line commented as `your action` to do something when the back key is pressed in the `hello.html` page.
+In this case, it should return to the main page of the Web application. The code above is working on renderer process. However, the API `loadURL` you used before to change the page of current window is working at browser process. The browser process should know when the `back key` is pressed.
+
+There is a set of APIs for messaging system between those 2 process in the Addon APIs.
+You have to decide the name of the message as you want. In this example, you will use `backkey` simply.
+Then, in `hello.html`, it should send a message `backkey` to browser process when the back key is pressed. You can replace `your action` above with the code below:
+
```

Comment:
Suggest rephrasing to:

You can add your code in the `try` block  to enable the Back key functionality in the `hello.html` page. In this case, it must return to the main page of the Web application. The preceding code is working on the renderer process. However, the API `loadURL` that you used earlier to change the page of the current window is working at the browser process. The browser process must know when the Back key is pressed.

There is a set of APIs for the messaging system between the two processes in the Addon APIs. You must specify the name of the message as you want. In this example, you can use `backkey` as the name. Then, in `hello.html`, it must send a message `backkey` to the browser process when the back key is pressed. You can replace your action using the following code:

Please verify for technical accuracy.

Reply:
done